src/styles.css

```diff
@@ -360,6 +357,16 @@
   margin-bottom: 0;
 }
 
+/* Select Instructions */
+.cio-select-question-instructions {
+  text-align: center;
+  font-size: 1.125rem;
+  color: var(--gray-dust-100);
+  line-height: 150%;
+  margin-top: 2.75rem;
+  margin-bottom: -1rem;
```

Comment:
This is to adjust the space between description and options in both single select and multi select. Let me know if negative margins would cause an issue